README.md

```diff
@@ -2458,6 +2458,179 @@ You can query multiple elements, for example you can add the section with the im
 ```
  <Debugging query="product=Photoshop&image=test" />
 ```
+# Get Credential 
+
+The Get Credential component allows you to easily generate credentials and access code samples directly from the developer website. Child component should only work if it's inside the parent GetCredential component 
+
+Use `credentialType`(optional) prop to specify the credential type . Defaults to `apiKey`.
+
+Use `service`(optional) prop to define the Adobe Product & Service.The preset is `CCEmbedCompanionAPI`.
+
+Use `className` to customize the component **at your own risk.**.
```

Comment:
I notice that in our example of the GetCredentialExternal.js actually have a className but was never really used in the css file.  We may want to remove that.  Also, include an explaination that they can just use css with className to customize the components to their desired style.